tachyon_core/src/error.rs

```diff
@@ -15,6 +15,8 @@ pub enum TachyonErr {
     ConnectionErr(#[from] ConnectionErr),
     #[error(transparent)]
     QueryErr(#[from] QueryErr),
+    #[error(transparent)]
+    InserterErr(#[from] WriterErr),
```

Comment:
Can we call this variant `WriterErr` to be consistent?

Reply:
Does WriterErr make sense if it comes from the inserter? I feel like this distinction makes sense

Reply:
Changed to WriterErr.